oat-sa/extension-tao-resource-workflow
# fix: When state is not existing set initial state for resource

This fix will reset the state for a resource when a resource has been imported from another instance that also contains the taoResourceWorkflow extension installed.

How to reproduce:
- Ensure your instance has taoResourceWorkflow installed
- Create item with content
- Export Item
- Open another environment with taoResourceWorkflow installed
- Import item
- Ensure the item has initially stated and defined in the workflow
